bindings/python/htcondor2/_submit.py

```diff
@@ -257,10 +270,16 @@ def setQArgs(self, args : str):
         Set the arguments to the queue statement.  These arguments replace
         the arguments, if any, passed to the original constructor.
 
-        :param args:  The queue arguments.
+        :param args:  The arguments.  May start with "queue ".
         '''
         if not isinstance(args, str):
             raise TypeError("args must be a string")
+
+        queue = args[0:5]
+        if queue.casefold() == "queue".casefold():
+            if args[5] == ' ':
```

Comment:
I think this might be a bit clearer/safer?
```suggestion
        if args.casefold().startswith("queue") and len(args) > 5 and args[5] == " ":
```